.eslintrc.js

```diff
@@ -0,0 +1 @@
+/Users/jackrans/Documents/hapi-riot/node_modules/goodparts/.eslintrc.js
```

Comment:
Just a quick question on this file, should it be on this PR as it contains your local path?